src/main/java/lotto/model/Revenue.java

```diff
@@ -0,0 +1,16 @@
+package lotto.model;
+
+import java.util.List;
+
+import lotto.policy.PrizeMoneyPolicy;
+
+public record Revenue(int budget) {
+
+    public double rateOfReturn(List<PrizeMoneyPolicy> ranks) {
+        long totalPrizeMoney = ranks.stream()
+                .map(PrizeMoneyPolicy::getPriceMoney)
+                .reduce(0L, Long::sum);
+
+        return ((double) totalPrizeMoney / budget) * 100;
```

Comment:
```suggestion
        return (double) totalPrizeMoney / budget * 100;
```
괄호 한 쌍 제거해도 결과는 같지 않을까요?

Reply:
저도 같은 결과여도 확실하게 보이기 위해서 넣을때가 있는데 빼는게 맞는걸까요? 🤔

Reply:
> 괄호 한 쌍 제거해도 결과는 같지 않을까요?

결과가 같습니다ㅎㅎ
근데 가로를 넣으면 사람이 볼때 더 이해하기 쉬울거 같아서 넣었어요!
너무 복잡한 가로넣기가 아니면 적당히 넣어줘도 문제 없을거 같다고 생각합니다ㅎㅎ